cinema/serializers.py

```diff
@@ -22,3 +22,63 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class ActorSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    first_name = serializers.CharField(max_length=255, required=True)
+    last_name = serializers.CharField(max_length=255, required=True)
```

Comment:
You don`t need to use required=True since it's default value

Reply:
Done!